forum/static/style.css

```diff
@@ -1,3 +1,49 @@
+@media(prefers-color-scheme: light){
+	body {
+		color: #888;
+		background: white;
+	}
+	.user_action > a {
+		color: white;
+	}
+	.smalltext{
+		color: black;
+	}
+	#footer {
+		background: white;
+	}
+	a {
+		color: blue
+	}
+}
+
+@media(prefers-color-scheme: dark){
+	body {
+		color: #888;
+		background: black;
+	}
+	.user_action > a {
+		color: #ddd;
+	}
+	.smalltext{
+		color: #ddd;
+	}
+	#footer {
+		background: darkslategray;
+	}
+	a {
```

Comment:
We need `a:visited` to separate clicked links from the new ones. I'd suggest adding:

```
a:visited {
	color: #be31be;
}
```